extensions/objects/models/objects.py

```diff
@@ -834,12 +834,25 @@ class Units(BaseObject):
     # Validation of the units is performed only in the frontend using math.js.
     # math.js is not available in the backend.
 
-    description = 'A units dict representation.'
-    default_value = {}
+    description = 'A list of unit dict components.'
+    default_value = []
 
     SCHEMA = {
-        'type': 'dict',
-        'properties': []
+        'type': 'list',
+        'items': {
+            'type': 'dict',
+            'properties': [{
+                'name': 'unit',
+                'schema': {
+                    'type': 'unicode'
+                }
+            }, {
+                'name': 'exp',
+                'schema': {
+                    'type': 'int'
+                }
+            }]
+        }
```

Comment:
@seanlip, just want to inform you of the new units schema.

Reply:
Can we change exp to exponent?

Otherwise LGTM, thanks.